release/preview/ubi83/meta.json

```
@@ -0,0 +1,18 @@
+{
+    "IsLinux" : true,
+    "UseLinuxVersion": false,
+    "PackageFormat": "powershell${previewTag}-${PS_VERSION}-1.rh.x86_64.rpm",
+    "SkipGssNtlmSspTests": true,
+    "osVersion": "RedHat Universal Base Image 8.3",
```

We can't use RedHat when describing these images.

```suggestion
    "osVersion": "Universal Base Image 8.3",
```